src/testMod/resources/data/advancedperipheralstest/computer/tests/peripheraltest.environment.lua

```diff
@@ -0,0 +1,5 @@
+detector = peripheral.find("environmentDetector")
+test.eq(detector ~= nil, true, "Peripheral not found")
```

Comment:
not sure if the method exists but I believe `test.neq(detector, nil` will better than `test.eq(x ~= nil`